catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswQueryFactory.java

```diff
@@ -388,6 +386,15 @@ private Filter parseFilter(FilterType filterType) throws CswException {
     return (Filter) parseJaxB(filterElement);
   }
 
+  private QueryRequest transformQuery(QueryRequest request, String typeName) {
```

Comment:
❓ This looks the same as the guts of the collection-based version. Is there any reason not to refactor the overloaded method to use this code?